modules/opencensus/pom.xml

```diff
@@ -100,14 +100,12 @@
             <groupId>io.opencensus</groupId>
             <artifactId>opencensus-exporter-stats-prometheus</artifactId>
             <version>${opencensus.version}</version>
-            <scope>test</scope>
```

Comment:
Can we use the runtime scope for exporters like zipkin, prometheus?

Reply:
Yes, changed the scope, thanks. Libs not needed for compilation, only for runtime.

Reply:
But what about zipkin exporter can we fix it too?

Reply:
But what about zipkin exporter can we fix it too?

Reply:
For now opencensus-exporter-trace-zipkin scope is test in AI, and my goal is to make metrics work, not traces. So I suggest to leave it as is for now for it.